docs/snippets/defaults_channel_mapping.mdx

```diff
@@ -35,4 +35,39 @@ default_platform_ad_format_by_channel:
   audio:    30s Audio
   app:      Interstitial - 1080x1920 Banner
   web:      Leaderboard - 728x90 Banner
+default_platform_ad_format_by_channel:
+  dooh:     Landscape - 1920x1080 Image
+  social:   Sponsored Post - 1080x1920 Image
+  ctv-bvod: 15s Video
+  audio:    30s Audio
+  app:      Interstitial - 1080x1920 Banner
+  web:      Leaderboard - 970x250 Banner
+default_property_average_imps_per_session_by_channel:
+  dooh:     1
+  social:   1
+  ctv-bvod: 1
+  audio:    1
+  app:      1
+  web:      1
+default_property_ad_funded_percentage_by_channel:
+  dooh:     1
+  social:   1
+  ctv-bvod: 1
+  audio:    1
+  app:      1
+  web:      1
+default_average_seconds_per_session_excluding_ads_by_channel:
+  dooh:     1
+  social:   1
+  ctv-bvod: 1
+  audio:    1
+  app:      1
+  web:      1
+default_average_data_kb_per_session_seconds_excluding_ads_by_channel:
```

Comment:
- [ ] populate defaults 

Comments
- I think for CTV/audio this comes from the average bitrate, probably. For app-based channels let's pull an 80th percentile number based on what we have from data.ai... I think we could probably get web from a sampling of archive.org if we want, or can take 80th percentile of our database? This one would be helped with Sincera data prob.